src/components/MetaHeadEmbed.tsx

```diff
@@ -66,8 +66,14 @@ export interface MetaEmbedProps {
   /** Webpage description. Should be less than 160 characters. */
   description: string;
 
+  /**
+   * Enable canonical even if the canonicalUrl is empty.
+   * Mainly used for homepages, where the canonicalPath is empty.
+   */
+  disableCanonical?: boolean;
+
   /** Canonical URL of your webpage that will be used as its default app URL. */
-  canonicalUrl?: string;
+  canonicalPath?: string;
```

Comment:
Wondering if this is needed? Can probably just use `pagePath`?

Reply:
Removed and added pageUrl taking the pagePath